Managers/FrameMan.cpp

```diff
@@ -1,2651 +1,1391 @@
+				blit(m_pNetworkBackBufferIntermediateGUI8[m_NetworkFrameCurrent][i], m_pNetworkBackBufferFinalGUI8[m_NetworkFrameCurrent][i], 0, 0, 0, 0, m_pNetworkBackBufferFinalGUI8[m_NetworkFrameCurrent][i]->w, m_pNetworkBackBufferFinalGUI8[m_NetworkFrameCurrent][i]->h);
+				m_NetworkBitmapIsLocked[i] = false;
+
+				// Draw all player's screen into one
+				if (g_UInputMan.KeyHeld(KEY_5)) {
+					stretch_blit(m_pNetworkBackBufferFinal8[m_NetworkFrameCurrent][i], m_pBackBuffer8, 0, 0, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][i]->w, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][i]->h, dx, dy, dw, dh);
+				}
+			}
+
+#if defined DEBUG_BUILD || defined MIN_DEBUG_BUILD
+			if (g_UInputMan.KeyHeld(KEY_1)) {
+				stretch_blit(m_pNetworkBackBufferFinal8[0][0], m_pBackBuffer8, 0, 0, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][0]->w, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][0]->h, 0, 0, m_pBackBuffer8->w, m_pBackBuffer8->h);
+			}
+			if (g_UInputMan.KeyHeld(KEY_2)) {
+				stretch_blit(m_pNetworkBackBufferFinal8[1][0], m_pBackBuffer8, 0, 0, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][1]->w, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][1]->h, 0, 0, m_pBackBuffer8->w, m_pBackBuffer8->h);
+			}
+			if (g_UInputMan.KeyHeld(KEY_3)) {
+				stretch_blit(m_pNetworkBackBufferFinal8[m_NetworkFrameReady][2], m_pBackBuffer8, 0, 0, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][2]->w, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][2]->h, 0, 0, m_pBackBuffer8->w, m_pBackBuffer8->h);
+			}
+			if (g_UInputMan.KeyHeld(KEY_4)) {
+				stretch_blit(m_pNetworkBackBufferFinal8[m_NetworkFrameReady][3], m_pBackBuffer8, 0, 0, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][3]->w, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][3]->h, 0, 0, m_pBackBuffer8->w, m_pBackBuffer8->h);
+			}
+#endif
+
+			// Rendering complete, we can finally mark current frame as ready
+			// This is needed to make rendering look totally atomic for the server pulling data in separate threads
+			m_NetworkFrameReady = m_NetworkFrameCurrent;
+			m_NetworkFrameCurrent = (m_NetworkFrameCurrent == 0) ? 1 : 0;
+
+			/*for (int i = 0; i < c_MaxScreenCount; i++)
 			{
-				// Oops, failed to set the resolution specified in the setting file, so default to a safe one instead
-				allegro_message("Unable to set specified graphics mode because: %s!\n\nNow trying to default back to VGA...", allegro_error);
-				if (set_gfx_mode(m_Fullscreen ? GFX_AUTODETECT_FULLSCREEN : GFX_AUTODETECT_WINDOWED, 640, 480, 0, 0) != 0)
-				{
+				m_NetworkBitmapIsLocked[i] = true;
+				blit(m_pBackBuffer8, m_pNetworkBackBuffer8[i], 0, 0, 0, 0, m_pBackBuffer8->w, m_pBackBuffer8->h);
+				m_NetworkBitmapIsLocked[i] = false;
+			}*/
+		}
+
+		// Do post-processing effects, if applicable and enabled
+		if (g_InActivity) { g_PostProcessMan.PostProcess(); }
+
+		// Draw the console on top of everything
+		//if (FlippingWith32BPP()) { g_ConsoleMan.Draw(m_pBackBuffer32); }
```

Comment:
Kill the commented code, it's old dead junk. I'd say keep the comment though, since it explains that the console is drawn on top of everything.